src/runtime/mod.rs

```diff
@@ -810,13 +818,13 @@ mod test {
     async fn load_routes_from_wasm() {
         let tf = write_temp_wat(ROUTES_WAT).expect("created tempfile");
         let watfile = tf.path();
+        let urlish = format!("file:{}", watfile.to_string_lossy().to_string());
```

Comment:
Nit: Do you need to `to_string()` here? Based on what I see in the docs, `Display` is implemented for `Cow` when `T` implements `Display`